src/main/java/teammates/storage/search/SearchManager.java

```diff
@@ -17,16 +28,52 @@
 
     private static final Logger log = Logger.getLogger();
 
+    private static final String ERROR_DELETE_DOCUMENT =
+            "Failed to delete document(s) %s in Solr. Root cause: %s ";
+    private static final String ERROR_SEARCH_DOCUMENT =
+            "Failed to search for document(s) %s from Solr. Root cause: %s ";
+    private static final String ERROR_SEARCH_NOT_IMPLEMENTED =
+            "Search service is not implemented";
+    private static final String ERROR_PUT_DOCUMENT =
+            "Failed to put document(s) %s into Solr. Root cause: %s ";
+    private static final String STUDENT_COLLECTION_NAME = "students";
+    private static final String INSTRUCTOR_COLLECTION_NAME = "instructors";
+
     private String searchServiceHost;
+    private HttpSolrClient client;
 
     public SearchManager(String searchServiceHost) {
         this.searchServiceHost = searchServiceHost;
+        if (isSearchServiceActive()) {
+            this.client = new HttpSolrClient.Builder(this.searchServiceHost).build();
+        }
     }
 
     private boolean isSearchServiceActive() {
         return !StringHelper.isEmpty(searchServiceHost);
     }
 
+    /**
+     * Reset the data for all collections in search server.
+     */
+    public void resetCollections() {
```

Comment:
This is a dangerous method if mistakenly called during production. Do we really need this? Test data should already be refreshed when using `removeAndRestoreDataBundle`

Reply:
Since now we need reset the collections in Solr after every component tests to allow clean data, perhaps we can move this method to a test helper class, so that it can only be called when running component tests?